parm/gefs/post_itag_gefs

```diff
@@ -0,0 +1,7 @@
+&MODEL_INPUTS
```

Comment:
can you name this the same way as gfs; e.g. `post_itag_gefs128`?

Reply:
@aerorahul Updated.